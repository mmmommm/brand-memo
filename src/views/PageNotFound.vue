<template>
  <pageNotFound/>
</template>
<script lang="ts">
import pageNotFound from '@/components/organisms/page-not-found.vue'
import { Component, Vue } from 'vue-property-decorator';
@Component({
  components: {
    pageNotFound
  }
})
export default class PageNotFound extends Vue {}
</script>