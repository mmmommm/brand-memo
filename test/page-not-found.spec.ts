import PageNotFound from '@/views/page-not-found.vue'
import Vuetify from 'vuetify'

import { createLocalVue, shallowMount } from "@vue/test-utils"

describe('page-not-found.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
    
  })
  const wrapper = shallowMount(PageNotFound, {
    localVue,
    vuetify,
  })
  it('<PageNotFound />', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})