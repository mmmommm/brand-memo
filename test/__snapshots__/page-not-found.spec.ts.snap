// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page-not-found.vue snapshot test 1`] = `<pagenotfound-stub />`;
